routers/web/repo/issue.go

```diff
@@ -1144,7 +1144,35 @@ func ValidateRepoMetas(ctx *context.Context, form forms.CreateIssueForm, isPull
 		assigneeIDs = append(assigneeIDs, form.AssigneeID)
 	}
 
-	return labelIDs, assigneeIDs, milestoneID, form.ProjectID
+	// Check reviewers
+	var reviewerIDs []int64
```

Comment:
Maybe we can check `isPull` to detect whether we need to check reviewer IDs or drop them.